core/app_modules/auth.js

```diff
@@ -56,12 +56,18 @@ module.exports = {
 
     redisClient.sadd([token, obj.username]);
     redisClient.expire(token, 43200);
-    client.query("INSERT INTO users(netid) VALUES ($1);", [obj.username] );
-  },
+    var is_user = yield check_id(obj.username);
+    if (is_user){
+      client.query("UPDATE users set email = $2 WHERE netid = $1;", [obj.username, obj.attributes.emailAddress]);
```

Comment:
What should we do when someone checks something out and we want to take their email address? Should we assume that it's up-to-date in their BYU information? I assume these people probably won't be logging in very often anyway, so this should be fine.

I say go ahead and merge. I don't think we'll run into any problems. LGTM